src/sys/mod.rs

```diff
@@ -226,3 +226,7 @@ feature! {
     #![feature = "time"]
     pub mod timer;
 }
+
+/// Fast user-space locking.
+#[cfg(any(target_os = "android", target_os = "linux"))]
+pub mod futex;
```

Comment:
We should probably put this behind a feature (see #1611).

Reply:
Added a `futex` feature.